src/Logary/DataModel.fs

```diff
@@ -1158,19 +1158,33 @@ module Message =
   [<CompiledName "EventFatalFormat">]
   let eventFatalf fmt = Printf.kprintf (event LogLevel.Fatal) fmt
 
+  open Logary.Utils.FsMessageTemplates
+
+  /// A destructuring strategy for FsMessageTemplates which simply treats 
+  /// everything as a 'Scalar' object which can later be handled by Logary
+  /// as `Field (Value.ofObject o, None)`
+  let internal destructureEverythingAsScalarObject : Destructurer =
+    fun request -> TemplatePropertyValue.ScalarValue request.Value
+  
+  let internal captureNamesAndValuesAsScalars (t: Template) (args: obj[]) = 
+    let selfLogByIgnoring : SelfLogger = fun (t, v) -> ()
```

Comment:
I am considering ignoring this issue in the short term, to get this moving along. What do you think @haf ?